include/dfu/pcd.h

```diff
@@ -87,8 +68,10 @@ int pcd_network_core_update(const void *src_addr, size_t len);
 int pcd_network_core_update_initiate(const void *src_addr, size_t len);
 
 /** @brief Lock the RAM section used for IPC with the network core bootloader.
+ *
+ * @param lock_conf Lock configuration until next SoC reset.
  */
-void pcd_lock_ram(void);
+void pcd_lock_ram(bool lock_conf);
```

Comment:
This change is not described in the commit message, is it intended to be in this commit?

Also the commit message does not have an NCSDK- jira reference, I thought those were mandatory?

Reply:
It is intended to be here. It is called from MCUboot: https://github.com/nrfconnect/sdk-mcuboot/pull/330/files#diff-a29d68e01a11ae09e8ee9cce6312d91783dddf32ee56a3ffc09e09e04fb9c259R604-R608